doc/source/api.rst

```diff
@@ -1870,8 +1870,52 @@ Methods
     Timedelta.to_timedelta64
     Timedelta.total_seconds
 
+.. _api.frequencies:
+
+Frequencies
+-----------
+
+.. currentmodule:: pandas.tseries.frequencies
+
+
+.. autosummary::
+   :toctree: generated/
+
+   to_offset
+
+.. _api.offsets:
+
+Offsets
+-------
+
+.. currentmodule:: pandas.tseries.offsets
+
+.. autosummary::
+   :toctree: generated/
+
+   DateOffset
```

Comment:
these weren't anywhere :< (not the complete list as didn't put up businesss ones).....